tests/test_parser.py

```diff
@@ -1,53 +1,138 @@
+# coding: utf-8
+
 from __future__ import unicode_literals
 import unittest
 
 import twingly_search
 
 class ParserTest(unittest.TestCase):
+    def get_fixture(self, fixture_name):
+        file_path = "./tests/fixtures/%s.xml" % fixture_name
+        fixture = open(file_path, 'r').read()
+        return fixture
+
+    def assert_blog_posts_equal(self, actual_post, expected_post):
```

Comment:
How about implementing `__eq__` on post instead. Seems to be pretty straight-foward:

```python
def __eq__(self, other):
        return (isinstance(other, self.__class__)
            and self.__dict__ == other.__dict__)
```

See: http://stackoverflow.com/questions/390250/elegant-ways-to-support-equivalence-equality-in-python-classes